CNA_Rules.md

```diff
@@ -433,6 +433,8 @@ CNAs are constrained to assigning CVE IDs to Vulnerabilities within their Scope
 
 4.2.17.7 CNAs MUST NOT assign CVE IDs for the sole reason that a Product is or has become EOL. See 4.1.13.
 
+4.2.17.8 If any CNA assigns a CVE ID for a Vulnerability in a supported Product or version, and the same Vulnerability exists in an EOL Product or version, no CNA SHALL assign a separate CVE ID for the same Vulnerability in the EOL Product or version.  Similar to 4.2.15, there is only one Vulnerability which MUST have one CVE ID.
```

Comment:
suggestion:

```diff
- no CNA SHALL assign
+ CNAs MUST NOT
```

the existing negation of the mandatory `SHALL` is awkward and potentially confusing

Reply:
another suggestion:

```diff
- 4.2.17.8 If any CNA
+ 4.2.17.8 If a CNA
```

for consistency in the doc

Reply:
Yes to both, thank you, I'll get the PR and Google doc syncd.